packages/@aws-cdk-testing/framework-integ/test/aws-iam/test/integ.user-import.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  App, Stack, CfnParameter,
+  aws_iam as iam,
+  CfnOutput,
+} from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+
+const app = new App();
+const stack = new Stack(app, 'dummy-stack');
+
+const userArn = 'arn:aws:iam::123456789012:user/OthersExternalIamUser';
+
+const userparam = new CfnParameter(stack, 'UserParameter', {
+  default: userArn,
+});
+
+const imported = iam.User.fromUserArn(stack, 'imported-user', userArn);
+const imported2 = iam.User.fromUserArn(stack, 'imported-user2', userparam.valueAsString );
```

Comment:
Might be worth adding the following just to make sure `Ref` imports still work properly:

```ts
User.fromUserArn(stack, 'LocalUser', new User(stack, 'User').userArn);
```